services/ui-auth/serverless.yml

```diff
@@ -41,6 +43,11 @@ resources:
         - Fn::Equals:
             - ""
             - ${self:custom.iamPermissionsBoundaryPolicy}
+    BackWithOkta:
```

Comment:
This is the Cloudformation conditional based on whether or not okta_metadata_url is blank or not.